core/src/main/scala/io/qbeast/core/model/ColumnsToIndexSelector.scala

```diff
@@ -0,0 +1,36 @@
+package io.qbeast.core.model
```

Comment:
I'm wondering why the license header checker is not capturing that this file is missing a license header altogether.

Reply:
Seems like the `core` package (`sbt project qbeastCore`) does not have LICENSE nor any way of creating them. I would open an issue and resolve it in another PR.

Reply:
Added headers on 883bed17